src/mujoco_system_interface.cpp

```diff
@@ -429,6 +429,11 @@ hardware_interface::CallbackReturn MujocoSystemInterface::on_init(const hardware
     return hardware_interface::CallbackReturn::FAILURE;
   }
 
+  // Disable the rangefinder flag at startup so that we don't get the yellow lines.
+  // We can still turn this on manually if desired.
+  auto rangefinder_flag_ptr = sim_->opt.flags + mjVIS_RANGEFINDER;
+  *rangefinder_flag_ptr = false;
```

Comment:
Aren't the flags an array?

```suggestion
  // Disable the rangefinder flag at startup so that we don't get the yellow lines.
  // We can still turn this on manually if desired.
  sim_->opt.flags[mjVIS_RANGEFINDER] = false;
```

Reply:
Yeah, for sure. I guess I was matching [mujoco's syntax](https://github.com/google-deepmind/mujoco/blob/36bed9a9da8cbd42a8dddd8ab3d586a51abedd7d/simulate/simulate.cc#L845), but it is easier to read in your change...so lets do that!